float64 gpst
float32[3] gb
float32[3] ab
float32[3] pos_std
float32[3] vel_std
float32[3] atti_std